packages/ui/src/utils/ClipboardManager.ts

```diff
@@ -36,8 +48,16 @@ export class ClipboardManager {
     try {
       const clipboardContents = await navigator.clipboard.read();
       for (const item of clipboardContents) {
-        if (item.types.includes(ClipboardManager.MIME_TYPE)) {
-          const blob = await item.getType(ClipboardManager.MIME_TYPE);
+        if (
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          (ClipboardItem as any).supports(ClipboardManager.KAOTO_MIME_TYPE) &&
```

Comment:
Why do we need to cast to `any` here?

Reply:
Following was the error from the `tsc`:
`error TS2339: Property 'supports' does not exist on type '{ new (items: Record<string, string | Blob | PromiseLike<string | Blob>>, options?: ClipboardItemOptions | undefined): ClipboardItem; prototype: ClipboardItem; }'.

27       if (ClipboardItem.supports(ClipboardManager.KAOTO_MIME_TYPE)) {
`

Reply:
Oh, that means that perhaps in some browsers, ClipboardItem might be of a different type, and there might not be a `supports` method in it.

Perhaps a safer way would be to check whether the clipboard item has the `support` method.